profile/README.md

```diff
@@ -23,7 +23,8 @@ Clear, transparent, and open communication is one of our keys to success.
 
 [![LinkedIn](https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/company/liquid-reply)
 [![Twitter](https://img.shields.io/badge/Twitter-%231DA1F2.svg?style=for-the-badge&logo=Twitter&logoColor=white)](https://twitter.com/LiquidReply)
-[![Website](https://img.shields.io/badge/Website-%2326A570.svg?style=for-the-badge&logo=readthedocs&logoColor=white)](https://liquidreply.net/)
+[![Website](https://img.shields.io/badge/Website-00b14e.svg?style=for-the-badge&logo=readthedocs&logoColor=white)](http://liquidreply.com)
+[![Blog](https://img.shields.io/badge/Blog-7D3C98.svg?style=for-the-badge&logo=readthedocs&logoColor=white)](https://liquidreply.net/)
```

Comment:
```suggestion
[![Company Blog](https://img.shields.io/badge/Blog-950EBA.svg?style=for-the-badge&logo=readthedocs&logoColor=white)](https://liquidreply.net)

```

Reply:
@leonardpahlke I have updated the color to the right color code, updated the alt text and removed a trailing slash. ;)